### DISCUSSION TITLE
Remove the coloured backgrounds on the playfield in osu!mainia argon skin.

### DISCUSSION THREAD

**Discussion by coltonisgod234** [ppy/osu#25344] [Ideas] [answered]
It makes it hard to read I think

**Comment by peppy** [comment, marked as answer]
I think this will be a toggle once we add more customisation to the skinning system.

**Reply by coltonisgod234** [reply]
should i keep this open until it gets added then? or do i close it now?

**Reply by Joehuu** [reply]
Should be kept open as it's not resolved/implemented.

**Comment by coltonisgod234** [comment]
I would say make this an opacity slider or something you can skin out.
